far/vmenu.hpp

```diff
@@ -142,11 +143,9 @@ struct MenuItemEx: menu_item
 	std::any ComplexUserData;
 	intptr_t SimpleUserData{};
 
-	size_t ShowPos{};
+	int HPos{}; // Positive: Indent; Negative: Hanging
```

Comment:
I went with `HorizontalPosition`. I thought about renaming all other identifiers containing "hpos" but decided against it because it becomes mouthful. Let me know if you like something else renamed.